pepper-apis/src/main/resources/db-changes/schema/DDP-6339-add-unique-validation-type.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="evgeniipr" id="20210802-add-unique-validation-type">
+        <insert tableName="validation_type">
+            <column name="validation_type_code" value="UNIQUE"/>
+        </insert>
+
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'UNIQUE')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'en')"/>
+            <column name="translation_text" value="Values should be unique."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
+            <column name="validation_type_id" valueComputed="(select validation_type_id from validation_type where validation_type_code = 'UNIQUE')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'ru')"/>
+            <column name="translation_text" value="Значения должны быть уникальны."/>
+        </insert>
+        <insert tableName="i18n_validation_msg_trans">
```

Comment:
@evgeniipr :
I agree we ccan remove fr/ru and add es
Rest overall looks good

Reply:
> @evgeniipr :
> I agree we ccan remove fr/ru and add es
> Rest overall looks good

@ssettipalli fixed (removed fr,ru), addes es